test/wpt/status/streams.json

```diff
@@ -16,6 +16,13 @@
   "readable-streams/cross-realm-crash.window.js": {
     "skip": "Browser-specific test"
   },
+  "readable-streams/from.any.js": {
+    "fail": {
+      "expected": [
+        "ReadableStream.from ignores a null @@asyncIterator"
```

Comment:
Do we need to fix this in a follow-up PR?

Reply:
@MattiasBuelens  I think we might need to address this issue. It seems like some basic error handling might be necessary. I'm also open to contributing to a follow-up PR to help resolve this.